toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -260,10 +276,16 @@ func createAllSRPMsWrapper(specsDir, distTag, buildDir, outDir, workerTar string
 	}
 
 	// If this is container build then the bind mounts will not have been created.
-	// Copy the chroot output to host output folder.
+	// Copy the chroot output and cached sources to host output folder.
 	if !buildpipeline.IsRegularBuild() {
 		srpmsInChroot := filepath.Join(chroot.RootDir(), outDir)
 		err = directory.CopyContents(srpmsInChroot, originalOutDir)
+		if err != nil {
+			return
+		}
+
+		cacheInChroot := filepath.Join(chroot.RootDir(), templateSrcConfig.localSourceCacheDir)
+		err = directory.CopyContents(cacheInChroot, originalCacheDir)
```

Comment:
What happens here in the default case where we didn't set the sources cache directory?

Reply:
Thanks, forgot to conditionalize this part. To answer your question, though, we'd eventually get an `ENOENT` error code from invoking the `stat` syscall (through a call to `os.MkdirAll` in `directory.CopyContents`) on an empty string.